### PR TITLE
Add getCartoColorPropsForItem function for collection of items using bold palette

### PR DESCRIPTION
**Pull request by vmilan**

### CODE REVIEW

**Comment by VictorVelarde** [comment]
image

Added method to re-iterate over the palette

File: packages/react-ui/storybook/stories/molecules/Avatar.stories.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import { Avatar, Grid, Box } from '@mui/material';
+import { Avatar, Grid, Box, useTheme } from '@mui/material';
 import makeStyles from '@mui/styles/makeStyles';
 import { Star } from '@mui/icons-material';
 import Typography from '../../../src/components/atoms/Typography';
+import { getCartoColorPropsForItem } from '@carto/react-ui';
```
[the remark this answers is not in the document]
Comment (vmilan) [reply packages/react-ui/storybook/stories/molecules/Avatar.stories.js:6]: I've tried at the beginning but didn't work for some errors don't recognizing the module... I'll give it another try

File: packages/react-ui/src/theme/carto-theme.js [hunk 2]
```diff
@@ -47,6 +47,8 @@ export const cartoThemeOptions = {
   },
   palette: {
     mode: 'light',
+    // Used by `getContrastText()` to maximize the contrast between the background and the text.
+    // https://mui.com/material-ui/customization/palette/#accessibility
```
Comment (VictorVelarde) [inline packages/react-ui/src/theme/carto-theme.js:51]: 👍🏻

**Review by VictorVelarde — APPROVED** [review]
Feel free to land it

**Pull request merged**